ebs_fields/fields.go

```diff
@@ -126,13 +126,16 @@ type GenerateVoucherFields struct {
 
 type VoucherCashOutFields struct {
 	CommonFields
-	PhoneNumber   string `json:"phoneNumber" binding:"required"`
-	VoucherNumber string `json:"voucherNumber" binding:"required"`
+	PhoneNumber   string  `json:"phoneNumber" binding:"required"`
+	VoucherNumber string  `json:"voucherNumber" binding:"required"`
+	TranAmount    float32 `json:"tranAmount" binding:"required" form:"tranAmount"`
 }
 
 type VoucherCashInFields struct {
 	CommonFields
-	VoucherNumber string `json:"voucherNumber" binding:"required"`
+	VoucherNumber string  `json:"voucherNumber" binding:"required"`
+	TranAmount    float32 `json:"tranAmount" binding:"required" form:"tranAmount"`
```

Comment:
in the voucher cash, the amount is also required

Reply:
I added it (TranAmount ) isn't that right ?

Reply:
Sorry, i mistake that.

Reply:
But, change the TranAmount to the struct `TransAmountFields`. `TransAmount` contains two fields: tranAmount and tranCurrencyCode, both are required. Also, make sure you are using the right `TranAmountFields` since Consumer have different semantics for that, i think it was called `ConsumerTransFields`, but I could be mistaken. Just use the same that is used in `Purchase` method